buildrelease.sh

```diff
@@ -48,13 +48,31 @@ done
 cd docs
 ./builddocs.sh root $projects
 
-echo "Release build complete for the following projects:"
+cd ..
+rm -rf releasedocs
+git clone https://github.com/GoogleCloudPlatform/google-cloud-dotnet.git releasedocs -c core.autocrlf=input
+cd releasedocs
+git checkout gh-pages
+
+for project in $projects
+do
+  rm -r ./docs/$project
```

Comment:
This will fail for new projects; use -rf instead to handle the directory not being present before.

Reply:
Done